### ISSUE TITLE
Stephanie - July 2019

### ISSUE THREAD

**Issue by bpbond** [PNNL-PREMIS/PREMIS-ghg#111]
### Field Work
- [ ] Finish elevating GCREW power lines (good job for L/M)
- [ ] Two rounds of CO2 and CH4 measurements, hopefully weeks 2 and 4, at least one with Norbert trailing Gru
- [x] July litter collection
- [x] Continue to catch up on litter sorting: at least 1 day with SULIs?
- [ ] Repaint/replace plot stakes
- [x] Nick, Nate, etc. measurements
- [x] Coordinate with Wenzhi and SULIs re sap flux expansion week
- [ ] We want to fill in all missing inventory species before fall

### Coding/data
- [ ] Data pipeline improvements listed in #110 and #106 
Script to handle litter data
- [ ] Upgrade real-time data summary graphs?
- [x] Enter handwritten data sheets from June 19 sorting
- [x] Enter litter data from 5 July sorting

### Mentoring Lillie #98 
- [x] Coordinate with BBL on R syllabus; lead Tuesday trainings
- [x] Be available for LH consultation re field sampling (July 11 with Uzay) and data

### Other
- [x] Start to plan out poster for August postgrad symposium
- [x] We really need to start doing some background reading for the planned Rs-sapflux combo paper. How about reading (not word for word, but definitely to the point of being able to summarize main issues, approach used, and findings) one paper per week from [this list](https://github.com/PNNL-PREMIS/PREMIS-ghg/issues/93#issuecomment-508899674)?

**Comment by bpbond** [comment]
Notes while I'm thinking about them:
* On Friday we got a almost all litter sorted from December 2018 and April 2019; one plot from the latter left to do--it's in the muffle furnace, on the upper right (bags are loose).
* Lillie has the data sheets from that day, as well as the handwritten data sheets from the day you and I were doing the R workshop. We need to get those entered and committed.
* On Saturday I got PVC stakes cut, drilled, and set in the ground to all the plots. It shouldn't take long to lift and tie the power cables.
